src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -52,26 +58,23 @@ public static class ConnectionBuilder
     /// </summary>
     public class ConnectionBuilder<TSourceType>
```

Comment:
Mark obsolete?

Reply:
This class is necessary within v4 so it cannot be marked obsolete.

Reply:
OK, then `//TODO: mark obsolete in v5` or post a issue about it.

Reply:
It will be removed in v5. I can add a comment to that effect

Reply:
```suggestion
    // TODO: Remove in v5
    public class ConnectionBuilder<TSourceType>
```